examples/multisig_plain/lib.rs

```diff
@@ -484,15 +484,20 @@ mod multisig_plain {
 
         /// Invoke a confirmed execution without getting its output.
         ///
+        /// If the transaction which is invoked transfers value, this value has
+        /// to be sent as payment with this call. The method will fail otherwise,
+        /// and the transaction would then be reverted.
+        ///
         /// Its return value indicates whether the called transaction was successful.
         /// This can be called by anyone.
-        #[ink(message)]
+        #[ink(message, payable)]
         pub fn invoke_transaction(
             &mut self,
             trans_id: TransactionId,
         ) -> Result<(), Error> {
```

Comment:
As a side note: There is another issue with this method. It internally uses
```
.map_err(|_| Error::TransactionFailed);
```
which currently would not revert the transaction. This will only happen once we have implemented error forwarding. 
Currently, in this error case, the transaction would be taken (i.e. removed) out of the `StorageStash`, but not be executed due to the error. It would then not be possible to give the execution another try, since the transaction has been removed.

Reply:
Yeah this is a very important point made and acts as yet another motivator to implement storage reversal on error for non-`ink-as-dependency` calls. In the meantime we are required to panic in these critical cases.